.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+python: '3.5'
+sudo: false
+env:
+  - TOXENV=py26
+  - TOXENV=py27
```

Comment:
why use python 3.5 as the python and then only test on 2.7 and the completely dead 2.6